react/__tests__/OrderBy.test.js

```diff
@@ -1,22 +1,54 @@
 /* eslint-env jest */
 import React from 'react'
-import { render } from 'test-utils'
+import { render, fireEvent } from '@vtex/test-tools/react'
 
 import OrderBy from '../OrderBy'
 import { setMobileState } from 'vtex.render-runtime'
 
 describe('<OrderBy />', () => {
   const renderComponent = customProps => mobile => {
     setMobileState(mobile)
+    const getLinkProps = jest.fn().mockImplementation(pageNumber => {
+      return {
+        page: `mockedPage${pageNumber}`,
+        params: 'mockedParams',
+      }
+    })
+
     const props = {
-      getLinkProps: () => ({ page: 'mokedPage' }),
       orderBy: 'OrderByTopSaleDESC',
+      getLinkProps,
       ...customProps,
     }
 
-    return render(<OrderBy {...props} />)
+    return { ...render(<OrderBy {...props} />), getLinkProps }
   }
 
+  it('should shown dropdown box on mobile mode', () => {
+    const { container } = renderComponent()(true)
+
+    let dropdownContainer = container.querySelector('.dropdownMobile > .dn')
+    const button = container.querySelector('button')
+    // Expect Dropdown container be redered with display none
```

Comment:
typo
```suggestion
    // Expect Dropdown container be rendered with display none
```

Reply:
Might use `.not.toBeVisible()`

https://github.com/gnapse/jest-dom#tobevisible

Reply:
Don't work `toBeVisible()` looks the visibility prop and we change `display` value.